frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -441,16 +478,19 @@ pods:
       node:
         goal: RUNNING
         cmd: |
-          set -e
+          set -ex
           export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
-          cp -v hdfs {{TASKCFG_ALL_HDFS_VERSION}}/bin/
           {{#SECURITY_KERBEROS_ENABLED}}
           export HADOOP_OPTS="-Djava.security.krb5.conf=$MESOS_SANDBOX/{{TASKCFG_ALL_HDFS_VERSION}}/etc/hadoop/krb5.conf $HADOOP_OPTS"
           {{#TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
           export HADOOP_OPTS="-Dsun.security.krb5.debug=true $HADOOP_OPTS"
           {{/TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
           {{/SECURITY_KERBEROS_ENABLED}}
           ./bootstrap
+          {{#SECURITY_KERBEROS_ENABLED}}
+          $JAVA_HOME/bin/java -jar krbtest.jar secrets-hdfs.keytab
+          echo "Keytab file fixed due to bug in HDFS version 3.2.0. See HADOOP-16283 {{HDFS_BUG_ISSUE_URI}}"
```

Comment:
Let's change this to a comment and move it one line up above the actual command. Something like
```
# Fix keytab file due to bug in HDFS version 3.2.0. See https://issues.apache.org/jira/browse/HADOOP-16283
$JAVA_HOME/bin/java -jar krbtest.jar secrets-hdfs.keytab
```

Reply:
Done